packages/packages.yaml.tmpl

```diff
@@ -321,59 +631,76 @@ components:
-            files: [bin/ng-monitoring-server] # output files.
-            flatten: true # will archive the files into the tarball file using flatten mode.
+          - name: "ng-monitoring-ee-{{ .Release.version }}-linux-{{ .Release.arch }}.tar.gz"
+            files: # output files.
+              - name: ng-monitoring-server
+                src:
+                  path: bin/ng-monitoring-server
+          - name: container image
+            type: image
+            dockerfile: config/release/dockerfiles/ng-monitoring/Dockerfile
+            files:
+              - name: ng-monitoring-server
+                src:
+                  path: bin/ng-monitoring-server
   ctl:
     desc: ctl components
     version: {{ .Release.version }} # segment version.
     routers: # match once.
       - description: interpret versions according to semantic version spec.
-        semver: # ref: https://carvel.dev/vendir/docs/v0.34.x/versions/
-          constraints: ">=7.1.0"
+        semver:
+          # ref: https://github.com/Masterminds/semver#checking-version-constraints
+          constraints: ">=7.1.0-0"
+          if: {{ semver.MatchConstraint ">=7.1.0-0" .Release.version }}
         os: [linux]
         arch: [amd64, arm64]
+        profile: [release]
         artifactory:
           repo: hub.pingcap.net/pingcap/ctl/package
           tags:
             {{- if .Git.sha }}
-            - {{ .Git.ref }}-{{ .Git.sha }}
+            - "{{ .Git.ref }}-{{ strings.Trunc 7 .Git.sha }}"
             {{- end }}
-            - {{ .Git.ref }}
+            - "{{ .Git.ref }}"
         artifacts:
           - name: "ctl-{{ .Release.version }}-linux-{{ .Release.arch }}.tar.gz"
             files:
               - name: binlogctl
                 src:
                   type: oci
-                  url: "hub.pingcap.net/pingcap/tidb-binlog:{{ .Git.ref }}"
+                  url: "hub.pingcap.net/pingcap/tidb-binlog/pacakge:{{ .Git.ref }}"
                   path: "tidb-binlog-{{ .Release.version }}-linux-{{ .Release.arch }}.tar.gz"
                   extract: true
                   extract_inner_path: binlogctl
               - name: cdc
                 src:
                   type: oci
-                  url: "hub.pingcap.net/pingcap/tiflow/package:{{ .Git.ref }}"
-                  path: "cdc-{{ .Release.version }}-linux-{{ .Release.arch }}.tar.gz"
+                  url: "hub.pingcap.net//tiflow/package:{{ .Git.ref }}"
+                  path: "cdc-ee-{{ .Release.version }}-linux-{{ .Release.arch }}.tar.gz"
                   extract: true
                   extract_inner_path: cdc
               - name: ctl
                 src:
                   type: http
-                  url: "https://tiup-mirrors.pingcap.com/tiup-v1.8.1-linux-{{ .Release.arch }}.tar.gz"
+                  url: "http://fileserver.pingcap.net/download/tiup/releases/v1.8.1/tiup-v1.8.1-linux-{{ .Release.arch }}.tar.gz"
```

Comment:
Currently we download it from internal file server, but where it comes from? we should use the origin source.

Reply:
It comes from https://github.com/pingcap/tiup/releases/tag/v1.8.1